openpype/tools/launcher/models.py

```diff
@@ -30,6 +32,13 @@ def __init__(self, dbcon, parent=None):
         # Cache of available actions
         self._registered_actions = list()
         self.items_by_id = {}
+        path = appdirs.user_data_dir("openpype", "pype_club")
```

Comment:
```suggestion
        path = appdirs.user_data_dir("openpype", "pypeclub")
```
otherwise it creates new directory. We use `pypeclub` elsewhere.